app/forms/nomination_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class NominationForm
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :local_authority_id, :school_id
+  validates :local_authority_id,
```

Comment:
I wonder if these validations are enough. In theory it's possible for these fields to be present but not belong to anything in our database. Unlikely though given our UI.

Reply:
I think these validations are here for user convenience, rather than for system integrity. The case you mention, of some user giving us a made up id via a hacky api call, will be stopped on Rails model integrity, or database integrity, I think.

Reply:
I agree. You'll get an ugly error message, but that's exactly what you deserve 😂

Reply:
Punitive justice to users misusing our service, that's the core principle of good UX 😂

Reply:
You're thinking about it the wrong way! It's positive reinforcement for users who use the service correctly!